src/main/java/org/sonarsource/sonarlint/ls/CommandManager.java

```diff
@@ -183,6 +212,22 @@ private void handleOpenRuleDescriptionFromCodeActionCommand(ExecuteCommandParams
     openRuleDescription(binding.orElse(null), ruleKey);
   }
 
+  private void handleBrowseTaintVulnerability(ExecuteCommandParams params) {
+    String taintUrl = getAsString(params.getArguments().get(0));
+    telemetry.taintVulnerabilitiesInvestigatedRemotely();
+    client.browseTo(taintUrl);
+  }
+
+  private void handleShowTaintVulnerabilityFlows(ExecuteCommandParams params) {
+    String issueKey = getAsString(params.getArguments().get(0));
+    String connectionId = getAsString(params.getArguments().get(1));
+    analysisManager.getTaintVulnerabilityByKey(issueKey)
+      .ifPresent(issue -> {
+        telemetry.taintVulnerabilitiesInvestigatedLocally();
+        client.showTaintVulnerability(ShowAllLocationsCommand.params(issue, connectionId, bindingManager::serverPathToFileUri));
+      });
```

Comment:
what does it mean if the taint vulnerability is not present ? is this possible ? Should we give some feedback to the user ?